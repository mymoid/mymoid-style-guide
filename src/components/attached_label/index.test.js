import React from 'react';
import Index from './index';
import ReactTestRenderer from 'react-test-renderer';

describe('<Index />', () => {
  it('should render Attached label', () => {
    const renderer = ReactTestRenderer.create(<Index />);

    expect(renderer.toJSON()).toMatchSnapshot();
  });
  it('should render above', () => {
    const renderer = ReactTestRenderer.create(<Index above />);

    expect(renderer.toJSON()).toMatchSnapshot();
  });
  it('should render below', () => {
    const renderer = ReactTestRenderer.create(<Index />);

    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
