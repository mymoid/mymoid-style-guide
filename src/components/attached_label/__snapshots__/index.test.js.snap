// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> should render Attached label 1`] = `
<div>
  <div
    className="sc-bwzfXH gJIwTT"
  >
    <span
      className="sc-htpNat kugwQL"
    />
  </div>
  <div
    className="sc-bdVaJa fwgixs"
  />
</div>
`;

exports[`<Index /> should render above 1`] = `
<div>
  <div
    className="sc-bdVaJa fwgixs"
  />
  <div
    className="sc-bwzfXH gJIwTT"
  >
    <span
      className="sc-htpNat kkUNoq"
    />
  </div>
</div>
`;

exports[`<Index /> should render below 1`] = `
<div>
  <div
    className="sc-bwzfXH gJIwTT"
  >
    <span
      className="sc-htpNat kugwQL"
    />
  </div>
  <div
    className="sc-bdVaJa fwgixs"
  />
</div>
`;
